src/dotty/tools/dotc/core/Contexts.scala

```diff
@@ -573,10 +578,15 @@ object Contexts {
     }
   }
 
+  val DocContext = new Key[DocBase]
   class DocBase {
```

Comment:
DocContext should be renamed to DocBase for consistency.
Both kinds of DocBase should also move to the dottydoc package, I think.